internal/access/credential.go

```diff
@@ -120,36 +121,34 @@ func hasMinimumCount(min int, password string, check func(rune) bool) bool {
 }
 
 func checkPasswordRequirements(db *gorm.DB, password string) error {
-	var errs []string
 	settings, err := data.GetSettings(db)
 	if err != nil {
 		return err
 	}
+	errs := make(validate.Error)
```

Comment:
I think it would be simpler to return an error from this function (similar to before), then wrap it in a validation error at the caller in a higher level.

Reply:
resolved in person